frontend/sentry.client.config.js

```diff
@@ -0,0 +1,19 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs"
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN
+
+Sentry.init({
+  dsn:
+    SENTRY_DSN ||
+    "https://b512f563700847e787e978ae1c15133c@o467221.ingest.sentry.io/6610580",
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1.0,
```

Comment:
I assume we don't want sampling rate to be 1.0 when we deploy this to www.flathub.org? We've got a generous plan from Sentry but I'd prefer not hitting its limits.

Reply:
I have no idea, will need to read up on it, that's what the setup generated.